.github/workflows/ci.yaml

```diff
@@ -0,0 +1,48 @@
+name: ci
+on:
+  push:
+    branches:
+      - master
+      - develop
+      - feat/dev-245-build-binaries
+    tags:
+      - "*"
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - macos-latest
+          - ubuntu-latest
+          - windows-latest
+        python:
+          - 3.5
+          - 3.6
+          - 3.7
+    env:
+      GDC_CLIENT_ZIP: "gdc-client.zip"
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python }}
+    - name: Run Tests
+      if: matrix.os != 'windows-latest'
```

Comment:
Tests fail on windows, after a very quick assessment it looks like fixtures use multiprocessing which is not windows friendly

Reply:
Is your vision for this that we migrate entirely from Travis to GitHub Actions? I wouldn't be opposed to that but I'd still bring it up with the rest of the team.

Reply:
is it upload by any chance? I added a threadpool executor 😆 maybe should've added processpool one

Reply:
I think he wants to reduce human involvement in building the actual images, where Travis can take care of that for us

Reply:
@ProfOak i imagine jenkins will be the travis replacement.

Reply:
I only wanted to build binaries and preserve the artifacts. Travis is still runs, I wasn't thinking about removing travis

Reply:
@ProfOak To be clearer, my goal was just to use actions build binaries for different os and preserve the artifacts here on github and possibly add them to the release. I added tests execution because it didn't increase the overall execution time by much, but yea I see your point as it makes travis somewhat redundant now. We can discuss that later if we want

Reply:
@philtom-ctds that was my impression as well. I just wanted to make sure.

One of the "on" events for GitHub actions is when a release tag is created. That might be easier to manage since we don't normally build binaries until we're getting ready to release. It would let another tool (Travis or Jenkins) handle the test execution part, too.

https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#onevent_nametypes